_posts/2019-07-23-extracting-fsspec-from-dask.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: Extracting fsspec from Dask
+author: Martin Durant
+tags: [IO]
+draft: true
+theme: twitter
+---
+{% include JB/setup %}
+
+
+Summary
+-------
+
+Over the past few years, Dask's IO capability has grown gradually and organically, to
+include a number of file-formats, and the ability to access data seamlessly on various
+remote/cloud data systems. This has been achieved through a number of sister packages
```

Comment:
Adding examples like HDFS or S3 here might make this more immediately concrete to readers.

```suggestion
remote/cloud data systems like HDFS, Amazon's S3, Google Cloud Storage, and others....
This has been achieved through a number of sister packages
```